ChiModules/LinearBoltzmannSolver/lbs_structs.h

```diff
@@ -41,10 +41,14 @@ struct Options
 
   int max_iterations = 1000;
   double tolerance    = 1e-8;
+
   bool use_precursors = false;
 
   bool save_angular_flux = false;
 
+  bool verbose_inner_iterations = true;
+  bool verbose_outer_iterations = true;
```

Comment:
I suppose all of these parameters can be changed by the user, correct? These are only default values, right?

Reply:
@ragusa That is correct. There are Lua routines that allow the user to change these in a Lua input file.